api/agentpb/agent.go

```diff
@@ -37,8 +37,10 @@ func (*SetStateRequest_AgentProcess) agentParams() {}
 func (*SetStateRequest_BuiltinAgent) agentParams() {}
 
 const (
-	mdID      = "pmm-agent-id"
-	mdVersion = "pmm-agent-version"
+	mdID            = "pmm-agent-id"
+	mdVersion       = "pmm-agent-version"
+	mdNodeID        = "pmm-agent-node-id"
+	mdServerVersion = "pmm-managed-version"
```

Comment:
I think we should do two things there:
1. We care about PMM Server version, not each individual component's version (they are the same). So let's use `pmm-server-version` for key.
2. Let's sync constant names with keys. For example, rename `mdID` to `mdAgentID`.

Reply:
1) How can I get `pmm-server-version`? PMM Server is a "Software System" which consists of `pmm-managed, Prometheus, Grafana, nginx, qan-api2, and qan-app` according to our architecture. There is no info about PMM Server version in `pmm-managed` basically. There is only info about `PMMVersion` and `pmm-managed` version.
2) Ok, I'll fix it. But I think this type of review suggestions have no sense as a conversation about where to add new private methods (after public one or before). At least at this moment. I mean we spend 2 times the longer time for code review than for implement something. If we all will doing code review with the same principle, it stops our development at all.

Reply:
>they are the same
PMM Server Version == PMM Version == pmm-managed version

2. Sure. But that PR was rejected not because of this, but due to incorrect working with metadata.